whisk/client.go

```diff
@@ -49,6 +50,8 @@ const (
     ExitWithErrorOnTimeout = true
     ExitWithSuccessOnTimeout = false
     DEFAULT_HTTP_TIMEOUT = 30
+    DEFAULT_ATTEMPTS = 5
+    DEFAULT_INTERVAL = 2 * time.Second
```

Comment:
Curious, why you selected 2 secs? I have seen a progressive approach where waiting 1 sec. on first iteration, 2 seconds on 2nd iteration, 4 sec. on 3rd iteration, and so forth. Really, my hope is that the HTTP request turnaround/round-trip time itself allows for success on a subsequent call. Do we really need to wait 2 seconds in addition to that?

Reply:
Also, 3 attempts seems to be an industry standard...